huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go

```diff
@@ -1263,3 +1331,46 @@ func getAvailableZones(cfg *config.Config, region string) ([]availablezones.Avai
 
 	return r.AvailableZones, nil
 }
+
+func handleMultiOperationsError(err error) (bool, error) {
+	if err == nil {
+		// The operation was executed successfully and does not need to be executed again.
+		return false, nil
+	}
+	// delete fail
+	if errCode, ok := err.(golangsdk.ErrDefault400); ok {
+		var apiError interface{}
+		if jsonErr := json.Unmarshal(errCode.Body, &apiError); jsonErr != nil {
+			return false, fmt.Errorf("unmarshal the response body failed: %s", jsonErr)
+		}
+
+		errorCode, errorCodeErr := jmespath.Search("error_code", apiError)
+		if errorCodeErr != nil {
+			return false, fmt.Errorf("error parse errorCode from response body: %s", errorCodeErr)
+		}
+
+		if errorCode.(string) == "DMS.00400026" {
+			// The operation failed to execute and needs to be executed again, because other operations are
+			// currently in progress.
+			return true, err
+		}
+	}
+	// unsubscribe fail
+	if errCode, ok := err.(golangsdk.ErrDefault400); ok {
+		var apiError interface{}
+		if jsonErr := json.Unmarshal(errCode.Body, &apiError); jsonErr != nil {
+			return false, fmt.Errorf("unmarshal the response body failed: %s", jsonErr)
+		}
+
+		errorCode, errorCodeErr := jmespath.Search("error_code", apiError)
+		if errorCodeErr != nil {
+			return false, fmt.Errorf("error parse errorCode from response body: %s", errorCodeErr)
+		}
+
+		// CBC.99003651: Another operation is being performed.
+		if errorCode == "CBC.99003651" {
+			return true, err
+		}
+	}
+	return false, err
```

Comment:
This is a repeated conditional judgment.

Reply:
done